pkg/webservice/handlers.go

```diff
@@ -39,6 +39,8 @@ import (
 	"github.com/apache/incubator-yunikorn-core/pkg/scheduler/objects"
 	"github.com/apache/incubator-yunikorn-core/pkg/webservice/dao"
 	"github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/si"
+
+	"github.com/gorilla/mux"
```

Comment:
This is licensed under BSD 3-Clause "New" or "Revised" License. I am not very familiar with the licensing aspects. Can you check if it is OK to use this? I think we need to mention this at least in some Notice file

Reply:
hi @kingamarton  I think this license is fine, and actually, this dependency is already been used by YK. See more info from the dependency doc in https://cwiki.apache.org/confluence/display/INCUBATOR/YuniKornProposal. Close this as resolved.